RocketApp/Domain/Sources/RocketsClient/Models/RocketDetail.swift

```diff
@@ -108,3 +110,59 @@ extension RocketDetail.Stage {
     case burnTime = "burn_time_sec"
   }
 }
+
+// swiftlint:disable identifier_name
+public extension RocketDetail {
+  enum RocketParameters: String {
+    case id, name, overview, height, diameter, mass, fuel_mass, first_stage, second_stage, first_flight, photos,
+      reusable, not_reusable, engines, tons_of_fuel, seconds_burn_time
+
+    public var name: LocalizedStringKey {
+      switch self {
+      case .name:
+        return self.rawValue.localizedStringKey
+      case .overview:
+        return self.rawValue.localizedStringKey
+      case .height:
+        return self.rawValue.localizedStringKey
+      case .diameter:
+        return self.rawValue.localizedStringKey
+      case .mass:
+        return self.rawValue.localizedStringKey
+      case .first_stage:
+        return self.rawValue.localizedStringKey
+      case .second_stage:
+        return self.rawValue.localizedStringKey
+      case .first_flight:
+        return self.rawValue.localizedStringKey
+      case .photos:
+        return self.rawValue.localizedStringKey
+      case .reusable:
+        return self.rawValue.localizedStringKey
+      case .not_reusable:
+        return self.rawValue.localizedStringKey
+      case .engines:
+        return self.rawValue.localizedStringKey
+      case .tons_of_fuel:
+        return self.rawValue.localizedStringKey
+      case .seconds_burn_time:
+        return self.rawValue.localizedStringKey
+      default:
+        return "Undefined"
+      }
+    }
+
+    public func detail(rocketDetail: RocketDetail, isUSMetrics: Bool) -> String {
```

Comment:
Tohle by tu asi bejt nemelo, rikal sem ti, ze modely by mely jen nest data, ne mit v sobe nejakou logiku. Mel bys na to mit nejakou dedikovanou vrstvu, `Formatter`. Muzes si tu udelat zatim TODO ale uz sem ti to rikal o tech modelech bez logiky...

Reply:
Udelam TODO.